package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vscode-kubernetes-tools",
     "displayName": "Kubernetes",
     "description": "Develop, deploy and debug Kubernetes applications",
-    "version": "1.0.9",
+    "version": "1.0.11",
```

Comment:
We will rev the version as part of a separate publish commit.  You may need to rev the version for testing but we should exclude it from the PR.

Reply:
This should also save having to update `package-lock.json` as part of the PR.